src/milia/api/io.cljs

```diff
@@ -64,11 +64,12 @@
               ["Accept" (or accept-header "application/json")]])))
 
 (defn get-xhr-io-response
-  [io-obj]
+  [io-obj & [{:keys [require-json?]}]]
   (try
     (.getResponseJson io-obj)
     (catch js/Error _
-      {:error (.getResponseText io-obj)})))
+      (let [text-response (.getResponseText io-obj)]
```

Comment:
This feels like the wrong place to put this, if `require-json?` is false, why should we parse for JSON at all?

I'd leave this as is and add a conditional on line 69, so that line looks like:

```clojure
(if require-json? .getResponseJson .getResponseText io-obj)
```